src/lib/apis/autoptic/index.ts

```diff
@@ -518,28 +518,32 @@ export const getListPQL = async () => {
 
 	let error = null;
 
-	const res = await fetch(`${AUTOPTIC_BASE_URL}/pqls/get_list_pql?endpoint_id=${endpoint_id}&serverURL=${encodeURIComponent(serverURL)}`, {
-		method: 'GET',
-		headers: {
-			'Content-Type': 'application/json',
-			// Authorization: `Bearer ${token}`
-		}
-	})
-		.then(async (res) => {
-			if (!res.ok) throw await res.json();
-			return res.json();
+	if (endpoint_id !== '' && serverURL !== '') {
```

Comment:
@jereolivieri deberías hacer una validación más sofisticada utilizando una RegEx. 
```suggestion
	if (typeof endpoint_id === 'string' && endpoint_id.trim() !== '' &&
    typeof serverURL === 'string' && serverURL.trim() !== '') {
    // Your code here
}

function isValidServerUrl(url) { const serverUrlRegex = /^(https?:\/\/)?([a-zA-Z0-9\-\.]+)(\.[a-zA-Z]{2,})(:[0-9]{1,5})?(\/.*)?$/; return serverUrlRegex.test(url); }
```

Reply:
The formatting and validation of the URL was done in another part of the application. So, is not necessary here.

Reply:
it’s not exactly the same regex validation, and due to how the code was previously written, it needs to be done this way